Update the instructions for compatibility with commander v2 (#27)
canonical/matter-docs: tutorial/pi-gpio-commander.md

sudo snap install matter-pi-gpio-commander
```

The application uses a
[custom-device](https://snapcraft.io/docs/custom-device-interface)
interface to access the GPIO. 
The interface should automatically connect upon installation. 
Let's verify that by looking at the snap's connections:
```console
$ sudo snap connections matter-pi-gpio-commander 
Interface      Plug                                      Slot                                      Notes
...
custom-device  matter-pi-gpio-commander:custom-gpio      matter-pi-gpio-commander:custom-gpio-dev  -
...
```

````{note}
On **Ubuntu Core**, the `custom-gpio` interface doesn't auto connect (See issue [#67](https://github.com/canonical/matter-pi-gpio-commander/issues/67#issuecomment-2180433237)).

To connect manually:
```bash
sudo snap connect matter-pi-gpio-commander:custom-gpio matter-pi-gpio-commander:custom-gpio-dev
```
````

### Configure the GPIO
Set the GPIO pin/line to 4:
```bash
sudo snap set matter-pi-gpio-commander gpio=4
```

> Apart from the GPIO pin, you may need to configure the GPIO chip.

Reviewer: ```suggestion
Apart from the GPIO pin, the GPIO chip may need to be configured.
```

```

Note that Thread communication requires a Thread Radio Co-Processor (RCP) and
an OpenThread Border Router (OTBR) agent enabling that communication over DBus.

To allow communication with the [OTBR Snap] for Thread management, connect the following interface:

```bash
sudo snap connect matter-pi-gpio-commander:otbr-dbus-wpan0 \
                  openthread-border-router:dbus-wpan0
```

> You may refer to [this guide](../how-to/otbr-on-ubuntu) setting up OTBR on Ubuntu.

Reviewer: ```suggestion
Refer to [this guide](../how-to/otbr-on-ubuntu) for setting up OTBR on Ubuntu.
```

> You may refer to [this guide](../how-to/otbr-on-ubuntu) setting up OTBR on Ubuntu.

Author: ```suggestion
You may refer to [this guide](../how-to/otbr-on-ubuntu) for setting up OTBR on Ubuntu.
```
